de.metas.banking/de.metas.banking.base/src/main/java/de/metas/banking/process/C_PaySelection_CreateFrom.java

```diff
@@ -104,6 +111,22 @@ else if (name.equals("C_BP_Group_ID"))
 		}
 	}
 
+	@Override
+	public ProcessPreconditionsResolution checkPreconditionsApplicable(final @NonNull IProcessPreconditionsContext context)
+	{
+		final IPaySelectionDAO paySelectionDAO = Services.get(IPaySelectionDAO.class);
+		final IPaymentDAO paymentDAO = Services.get(IPaymentDAO.class);
+
+		final I_C_PaySelection paySelection = paySelectionDAO.retrievePaySelectionById(context.getSingleSelectedRecordId());
+
+		if (paySelection.getDocStatus().equals(DocStatus.Drafted.getCode()))
```

Comment:
not 100% sure, but i think this won't work if you reactivate a pay selection. because then it is in "in progress".
So,
* so, use DocStatus.isDraftedOrInProgress()
* also make sure to avoid NPEs. I.e. use ofNullableCode 
* note: i guess that assuming the C_PaySelection record itself exists is OK

Reply:
Yes, it's not working if gets reactivated.

Reply:
agree with Tobi. It shall be as Tobi suggested